dashboard/client/src/type/serve.ts

```diff
@@ -84,12 +85,33 @@ export enum ServeDeploymentMode {
   FixedNumber = "FixedNumber",
 }
 
+// Keep in sync with HTTPProxyStatus in python/ray/serve/_private/common.py
+export enum ServeHTTPProxyStatus {
+  STARTING = "STARTING",
+  HEALTHY = "HEALTHY",
+  UNHEALTHY = "UNHEALTHY",
+}
+
+export type ServeHttpProxy = {
+  node_id: string;
+  node_ip: string;
+  actor_id: string;
+  actor_name: string;
+  status: ServeHTTPProxyStatus;
+  log_file_path: string | null;
```

Comment:
when is the log null - would it be when it's starting up?

Reply:
not sure. This is based on the backend api. @zcin ?

Reply:
Yeah the log file can be null for http proxy and replica actors before they start up.